src/jit/morph.cpp

```diff
@@ -2880,21 +2879,26 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
 
             nonStandardArgs.Add(arg, REG_VIRTUAL_STUB_PARAM);
         }
-        else if (call->gtCallType == CT_INDIRECT && call->gtCallCookie)
+        else
+#endif // !defined(_TARGET_X86_)
+        if (call->gtCallType == CT_INDIRECT && call->gtCallCookie)
         {
             assert(!call->IsUnmanaged());
 
-            // put cookie into R11
             GenTree* arg = call->gtCallCookie;
             noway_assert(arg != nullptr);
             call->gtCallCookie = nullptr;
 
             call->gtCallArgs = gtNewListNode(arg, call->gtCallArgs);
             numArgs++;
 
+            // x86 passes the cookie on the stack.
```

Comment:
nit: This is going to get screwed up by clang-format. Can you add a CLANG_FORMAT_COMMENT_COMMENT_ANCHOR?